### PR TITLE
Create a README for installing the Dev Sandbox in a development environment

### PR DESCRIPTION
**Pull request by rajivnathan**
Related issue: https://issues.redhat.com/browse/CRT-1028

PR changes:
- created a new README for instructions to run in a dev environment
- moved the list of required tools to its own doc to make it cleaner when following instructions from any of the READMEs
- added a new section to the main readme "Running in a Development Environment"

### CODE REVIEW

**Comment by rajivnathan** [comment]
Screenshot for waiting for approval:
image

**Comment by rajivnathan** [comment]
Login selection image:
image

**Review by alexeykazakov — APPROVED** [review]
Looks good!

File: dev_install.adoc
Comment (alexeykazakov) [inline dev_install.adoc:59]: Maybe worth mentioning how to get <usersignup name> and what it is. Just to make it clear that it's not the username.

Comment (rajivnathan) [reply dev_install.adoc:65]: Done https://github.com/codeready-toolchain/toolchain-e2e/pull/280/commits/8d38de5ee123984bca02d9ce764fbeccf823c7a3

File: dev_install.adoc
Comment (sbryzak) [inline dev_install.adoc:53]: Could we also include instructions for enabling auto-approval?

Comment (sbryzak) [reply dev_install.adoc:53]: haha, i should have read further down, disregard this :)

**Review by sbryzak — APPROVED** [review]
looks good!

**Comment by krishvoor** [comment]
Thanks for the PR @rajivnathan. 
/lgtm

**Pull request merged**
